quicklogic/common/toolchain_wrappers/conda_build_install_package.sh

```diff
@@ -24,10 +24,10 @@ CONDA_FLAGS="-y --override-channels -c defaults -c conda-forge"
 conda update $CONDA_FLAGS -q conda
 curl https://storage.googleapis.com/symbiflow-arch-defs-install/quicklogic-arch-defs-63c3d8f9.tar.gz --output arch.tar.gz
 tar -C $INSTALL_DIR -xvf arch.tar.gz && rm arch.tar.gz
-conda install $CONDA_FLAGS -c quicklogic-corp/label/ql yosys="0.8.0_0002_gc3b38fdc 20200901_073908" python=3.7
-conda install $CONDA_FLAGS -c quicklogic-corp/label/ql yosys-plugins="1.2.0_0009_g9ab211c 20201001_121833"
-conda install $CONDA_FLAGS -c quicklogic-corp/label/ql vtr="v8.0.0_rc2_2894_gdadca7ecf 20201008_140004"
-conda install $CONDA_FLAGS -c quicklogic-corp iverilog
+conda install $CONDA_FLAGS -c litex-hub/label/main symbiflow-yosys="0.9_0_g40d9e120 20210318_102115_libffi33"
```

Comment:
This should be `yosys` only, as per the environment.yml comment.

Reply:
reverted